src/components/about/about.module.css

```diff
@@ -0,0 +1,50 @@
+.about {
+  display: flex;
+  flex-direction: column;
+}
+
+.foto {
+  max-width: 512px;
+  height: 294px;
+  border-radius: 30px;
+  margin-bottom: 30px;
+  object-fit: cover;
+}
+
+.title {
+  font-size: 27px;
+  font-weight: 600;
+  color: #111;
+  margin-bottom: 30px;
+}
+
+.description {
+  font-size: 16px;
+  font-weight: 400;
+  color: #111;
```

Comment:
I'm not sure it's the right color from the design. It's hard to read 
![Screenshot 2024-04-21 at 21 05 46](https://github.com/IgorFromUA/fe-react-2024/assets/115080771/c8cae49c-f682-4308-ad4c-2caf25602f2d)

Reply:
please check all styles